api/serializers.py

```diff
@@ -1699,6 +1699,7 @@ class UserMeSerializer(UserSerializer):
     is_per_admin_for_regions = serializers.SerializerMethodField()
     is_per_admin_for_countries = serializers.SerializerMethodField()
     user_countries_regions = serializers.SerializerMethodField()
+    limit_access_to_guest = serializers.SerializerMethodField()
```

Comment:
Let's use source here
```suggestion
    limit_access_to_guest = serializers.SerializerMethodField(source='profile.limit_access_to_guest')
```